cli/cmd/admin/start.go

```diff
@@ -73,15 +74,22 @@ func StartCmd(cliCfg *config.Config) *cobra.Command {
 			}
 			defer adm.Close()
 
+			var deviceVerificationHost string
+			if conf.DeviceVerificationHost == "" {
+				deviceVerificationHost = server.DefaultDeviceVerificationHost
+			} else {
+				deviceVerificationHost = conf.DeviceVerificationHost
+			}
```

Comment:
We should set the default in the struct tag instead like:
```
DeviceVerificationHost string `default:"http://..." split_words:"true"
```
then can also remove the server.DefaultDeviceVerificationHost constant

Reply:
Also, the default should be the localhost/dev environment host. Then we'll override the env var in our prod deploys

Reply:
ok

Reply:
In the admin `README.md` I see `--api-url http://localhost:9090`, is this a mistake since 9090 is the default grpc port ?

Reply:
Not sure if I got the question right. The `--api-url` should point to the gRPC endpoint, not the HTTP endpoint.

Also the default `--api-url` in the CLI is `https://admin.rilldata.com` (since the CLI gets installed by users, so they shouldn't have to override it).

Reply:
During development for `login` it needs HTTP endpoint, where as for other cli cmds like create org it needs to use grpc endpoint so we cannot consistently use this flag across cli cmds. Thats my main confusion.

Reply:
I understand your point now. Since this only applies for `rill login`, for now, you can just pass `--api-url http://localhost:8080` for `rill login` in development and `--api-url http://localhost:9090` for other admin CLI commands. Then it should work, right?

Reply:
(In production, gRPC and REST will be on the same endpoint, `admin.rilldata.com`, it's only in prod that the ports are different.)

Reply:
sure